packit/api.py

```diff
@@ -1231,6 +1231,40 @@ def sync_release(
 
                 if fast_forward_merge_branches:
                     for ff_branch in fast_forward_merge_branches:
+                        logger.info(
+                            f"Syncing branch {ff_branch} defined in `fast_forward_merge_into`",
+                        )
+                        self.dg.refresh_specfile()
```

Comment:
I am  not sure this command (`refresh_specfile`) is really helpful. Later we remove any local change with the `switch_branch force=True`, right?

I think @nforro is working on fixing the above calls we are mimicking here (creat/update/switch). We should remember of fixing them also here later, right?